### ISSUE TITLE
Any method to using the same browser tab when clilck the Url of Link Mappings 

### ISSUE THREAD

**Issue by cumt10105** [algenty/grafana-flowcharting#263]
Hi Expert,
We have met an issue when using Url of Link Mappings, we want to config a url in chartflowing panel, when enduser click the object we linked, a new browser tab opened, shall we have way to using the same tab, just like  window.locationwindow.location.href. 
The configuration detail as following image
![image](https://user-images.githubusercontent.com/43465077/103847903-26dd5a00-50dc-11eb-9406-7f3f2bdc3823.png)

Our flowcharting version is 0.8.0

Any message will be appreciated.
Bing

**Comment by algenty** [comment]
i can't, link uses process from draw.io.

**Comment by udhayaas97** [comment]
Is any working solution for this?

**Comment by algenty** [comment]
no, have you try to put your url in draw.io to see the result ?

**Comment by pcukoo** [comment]
@udhayaas97 [udhayaas97](https://github.com/udhayaas97)

Can you share your solution?
